src/osd/OSD.cc

```diff
@@ -9210,6 +9212,11 @@ void OSD::handle_conf_change(const struct md_config_t *conf,
     op_tracker.set_history_size_and_duration(cct->_conf->osd_op_history_size,
                                              cct->_conf->osd_op_history_duration);
   }
+  if (changed.count("osd_op_history_slow_op_size") ||
+      changed.count("osd_op_history_slow_op_threshold")) {
+    op_tracker.set_history_size_and_duration(cct->_conf->osd_op_history_slow_op_size,
+                                             cct->_conf->osd_op_history_slow_op_threshold);
```

Comment:
this is calling the wrong setter

And can we do "recent_slow" instead of "slow"?  the historic ops are already the slowest ops ever; the difference here is that we're also tracking the most recent N slow ops